src/test/java/gumtree/spoon/TreeTest.java

```diff
@@ -542,7 +542,7 @@ public void test_getTree_labelExistsForAnnotations() {
 		Tree root = scanner.getTree(spoonType);
 
 		Tree annotationNode = root.getChild(0).getChild(0).getChild(2);
-		assertEquals("@java.lang.Override", annotationNode.getLabel());
+		assertEquals("java.lang.Override", annotationNode.getLabel());
```

Comment:
This change is related to https://github.com/SpoonLabs/gumtree-spoon-ast-diff/pull/221/files#diff-f779cea6d31aed1a7488d0b60806bb6cb5d6eb9bfb91aaa0b6fb80fef497b07bR143.

We want the qualified name of the type of the annotation not the whole string representation.